genie/database_to_staging.py

```diff
@@ -189,23 +190,35 @@ def redact_phi(
     # Moved to cannotReleaseHIPAA and withheld because the HIPAA
     # years would change every single year. (e.g. <1926, >1998 would be
     # inaccurate every year)
+    complete_redact = pd.Series([False] * len(clinicaldf))
     for col in interval_cols_to_redact:
-        to_redact, to_redactpeds = _to_redact_interval(clinicaldf[col])
-        clinicaldf.loc[to_redact, "BIRTH_YEAR"] = "cannotReleaseHIPAA"
-        clinicaldf.loc[to_redact, col] = ">32485"
-        clinicaldf.loc[to_redactpeds, "BIRTH_YEAR"] = "withheld"
-        clinicaldf.loc[to_redactpeds, col] = "<6570"
+        to_redact, to_redact_peds = _to_redact_interval(clinicaldf[col])
```

Comment:
This complicates things a little bit: https://sagebionetworks.slack.com/archives/C0350GLJ7M4/p1746489087447869?thread_ts=1746483192.246719&cid=C0350GLJ7M4. 

Basically if a patient contains fields that are to be redacted, every age variable that that patient across all samples need to be redacted